validation/authValidation.js

```diff
@@ -0,0 +1,104 @@
+const Joi = require('joi');
+
+module.exports = {
+  registration: Joi.object({
+    name: Joi.string().min(3).max(30).required().empty().messages({
+      'string.base': `name should be a type of text`,
+      'string.empty': `name cannot be an empty field`,
+      'string.min': `name should have a minimum length of 3`,
+      'any.required': `name is a required field`,
+    }),
+
+    email: Joi.string().min(11).max(50).required().empty().email().messages({
```

Comment:
also put email format validation
'string.email': 'Email id should be in correct format'